pkg/pub_worker/lib/src/analyze.dart

```diff
@@ -91,10 +95,30 @@ Future<void> analyze(Payload payload) async {
         }
       } catch (e, st) {
         _log.shout(
-          'failed to process ${payload.package} / ${p.version}',
+          'failed to process ${payload.package} / ${vtp.version}',
           e,
           st,
         );
+
+        String? logContent;
+        final logTxtFile = File(p.join(analysisTempDir.path, 'out', 'log.txt'));
+        if (logTxtFile.existsSync()) {
+          logContent = logTxtFile.readAsStringSync();
+        }
+
+        await _reportPackageSkipped(
+          client,
+          api,
+          payload.package,
+          vtp.version,
+          reason: [
+            'Processing of `${payload.package}/${vtp.version}` failed with the following error:\n$e',
+            st.toString(),
+            if (logContent != null) logContent,
+          ].join('\n\n'),
+        );
```

Comment:
I think the result of this will be that the package won't be retried with backoff.

Reply:
When pana fails, analysis should not be retried.

We only want to retry infrastructure failures.